### PR TITLE
Updating the French version of the Astar portal

### PR DESCRIPTION
**Pull request by Gunit2481**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository.

- Translation of the [Index.ts](https://github.com/AstarNetwork/astar-apps/blob/main/src/i18n/en-US/index.ts) file from the root `astar-apps/blob/main/src/i18n/en-US` into French
- Updating the [French Astar portal file](https://github.com/AstarNetwork/astar-apps/blob/main/src/i18n/fr/index.ts) from the root `astar-apps/blob/main/src/i18n/fr`
- Translation has been verified by the three french ambassadors

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**
- (ex: Add feature A)

**Fixes**
- (ex: Fix validation function)

**Changes**
- (ex: Change document B)

**To-dos**
> *Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Review by 0xRamz — APPROVED** [review]

File: src/i18n/fr/index.ts [hunk 1]
```diff
@@ -1,113 +1,551 @@
...
+  wallet3: 'Wallet 3',
+  alert: "Alerte",
+  max: 'Max',
+  native: "Natif",
+  evm: 'EVM',
+  wasm: 'WASM',
+  addressFormat: "Adresse du portefeuille {network}",
+  addressPlaceholder: "Adresse de destination {network}",
+  evmAddressPlaceholder: "Adresse EVM de destination",
+  ticker: 'Ticker',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:32]: Téléscripteur

ex: https://www.cairn.info/revue-reseaux1-2003-6-page-137.htm#pa2

File: src/i18n/fr/index.ts [hunk 2]
```diff
@@ -1,113 +1,551 @@
...
-    updateMetadata: 'Mettre à jour les métadonnées',
-    metadataAlreadyInstalled: 'Métadonnées déjà installées',
-    lightMode: 'Mode de jour',
-    darkMode: 'Mode de nuit',
+    updateMetadata: "Mettre à jour les métadonnées",
+    metadataAlreadyInstalled: "Métadonnées déjà installées",
+    lightMode: "Mode jour",
+    darkMode: "Mode nuit",
     dApps: 'dApps',
     dappStaking: 'dApp Staking',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:68]: Jalonnement dApps

ex: https://kriptomat.io/fr/crypto-monnaies/quest-ce-que-le-jalonnement-de-crypto-monnaies/

File: src/i18n/fr/index.ts [hunk 3]
```diff
@@ -1,113 +1,551 @@
...
-    contract: 'Contrat',
+    staking: 'Staking',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:69]: Jalonner

File: src/i18n/fr/index.ts [hunk 4]
```diff
@@ -1,113 +1,551 @@
...
+    forum: 'Forum',
+    github: 'Github',
+    docs: 'Docs',
+    settings: "Paramètres",
+    language: "Langue",
+    theme: "Thème",
+    close: "Fermer"
+  },
+  drawer: {
+    endpoint: 'Endpoint',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:108]: Point de terminaison

ex: https://m.interglot.com/en/fr/endpoint

File: src/i18n/fr/index.ts [hunk 5]
```diff
@@ -1,113 +1,551 @@
...
-    howToConnect: 'Apprendre à se connecter',
+    howToConnect: "Apprendre à se connecter"
+  },
+  updateWallet: {
+    getUpdatedWallet: "Vous n'avez pas encore mis à jour {value}?",
+    updateWallet: "Vous devrez mettre à jour {value} pour continuer. Une fois que vous l'aurez mis à jour vers la dernière version, allez-y et actualisez cette page",
+    updateExtension: "Mettre à jour l'extension {value}"
+  },
+  topMetric: {
+    build2earn: 'Build2Earn',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:134]: Construire pour gagner

File: src/i18n/fr/index.ts [hunk 6]
```diff
@@ -1,113 +1,551 @@
...
+    wayOfStaking: "Une manière innovante de faire du staking",
+    tvlInDapps: "TVL sur les dApps",
+    currentEra: "Ère actuelle",
+    currentStakersApr: "APR actuel pour les Stakers",
+    currentBlock: "Bloc actuel",
+    totalDapps: 'Total dApps'
```
Comment (gluneau) [inline src/i18n/fr/index.ts:140]: dApps au total

File: src/i18n/fr/index.ts [hunk 7]
```diff
@@ -1,113 +1,551 @@
...
+  },
+  myDapps: {
+    index: 'Index',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:143]: Indice

File: src/i18n/fr/index.ts [hunk 8]
```diff
@@ -1,113 +1,551 @@
...
+    dapps: 'dApps',
+    stakedAmount: "Montant staké",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:145]: Montant jalonné

File: src/i18n/fr/index.ts [hunk 9]
```diff
@@ -1,113 +1,551 @@
...
+    unbondingAmount: "Montant d'unbonding",
+    remainingEra: "Ère restante",
+    withdraw: "Retirer",
+    rebond: "Lier de nouveau",
+    totalEarned: "Total des gains",
+    manage: "Gérer",
+    add: "Ajouter",
+    unbond: 'Unbond',
+    rebondGuide: "Une fois que vos fonds seront reliés en staking, vous aurez besoin de 10era pour les libérer à nouveau.",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:154]: Une fois que vos fonds seront en jalonnement, vous aurez besoin de 10 ères pour les libérer à nouveau.

File: src/i18n/fr/index.ts [hunk 10]
```diff
@@ -1,113 +1,551 @@
...
+    dapps: 'dApps',
+    stakedAmount: "Montant staké",
+    unbondingAmount: "Montant d'unbonding",
+    remainingEra: "Ère restante",
+    withdraw: "Retirer",
+    rebond: "Lier de nouveau",
+    totalEarned: "Total des gains",
+    manage: "Gérer",
+    add: "Ajouter",
+    unbond: 'Unbond',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:153]: Délier

File: src/i18n/fr/index.ts [hunk 11]
```diff
@@ -1,113 +1,551 @@
...
+    currentEra: "Ère actuelle",
+    currentStakersApr: "APR actuel pour les Stakers",
+    currentBlock: "Bloc actuel",
+    totalDapps: 'Total dApps'
+  },
+  myDapps: {
+    index: 'Index',
+    dapps: 'dApps',
+    stakedAmount: "Montant staké",
+    unbondingAmount: "Montant d'unbonding",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:146]: Montant à délier

File: src/i18n/fr/index.ts [hunk 12]
```diff
@@ -1,113 +1,551 @@
...
+    manage: "Gérer",
+    add: "Ajouter",
+    unbond: 'Unbond',
+    rebondGuide: "Une fois que vos fonds seront reliés en staking, vous aurez besoin de 10era pour les libérer à nouveau.",
+    rebondTitle: "Le montant que vous souhaitez relier",
+    withdrawGuide: "Vos fonds sont disponibles pour être retirés.",
+    withdrawTitle: "Prêt à être retiré"
+  },
+  myReward: {
+    totalStaked: "Total Staké",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:160]: Total jalonné

File: src/i18n/fr/index.ts [hunk 13]
```diff
@@ -1,113 +1,551 @@
...
+    availableToClaim: "Disponible à réclamer",
+    era: "Ère",
+    claim: 'Claim',
+    restake: "Re-staker après avoir réclamé",
+    turnOff: "Désactiver",
+    totalEarned: "Total des gains (toujours)",
+    availableToClaimTip: "Le nombre d'ères qui est affiché ici est par dApp. Le nombre maximum d'ères que vous pouvez réclamer à la fois est de 50. Il se peut que vous deviez réclamer plusieurs fois si vous ne l'avez pas fait depuis longtemps.",
+    restakeTip: "En activant, vos récompenses seront automatiquement remises en staking lorsque vous ferez une réclamation."
```
Comment (gluneau) [inline src/i18n/fr/index.ts:168]: En activant, vos récompenses seront automatiquement remises en alonnement lorsque vous ferez une réclamation.

File: src/i18n/fr/index.ts [hunk 14]
```diff
@@ -1,113 +1,551 @@
...
   },
   dappStaking: {
+    myStaking: "Mon Staking",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:171]: Mon jalonnement

File: src/i18n/fr/index.ts [hunk 15]
```diff
@@ -1,113 +1,551 @@
...
+    myRewards: "Mes Récompenses",
+    unbonding: 'Unbonding',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:173]: Délier

File: src/i18n/fr/index.ts [hunk 16]
```diff
@@ -1,113 +1,551 @@
...
+    myDapps: "Mes dApps",
+    dappRegistered: "Félicitations!! Votre contrat a été approuvé. Veuillez soumettre les détails",
+    welcomeBanner: "Félicitations 🎉 Veuillez renseigner les informations de votre dApp pour donner aux utilisateurs une meilleure vue d'ensemble de votre application. Les données mises à jour apparaîtront sous peu sur la page de votre dApp.",
+    registerNow: "S'enregistrer maintenant",
+    transferableBalance: "Solde transférable",
     dappsStore: 'dApps Store',
     dappStaking: 'dApp Staking',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:180]: Jalonnement dApp

File: src/i18n/fr/index.ts [hunk 17]
```diff
@@ -1,113 +1,551 @@
...
   dappStaking: {
+    myStaking: "Mon Staking",
+    myRewards: "Mes Récompenses",
+    unbonding: 'Unbonding',
+    myDapps: "Mes dApps",
+    dappRegistered: "Félicitations!! Votre contrat a été approuvé. Veuillez soumettre les détails",
+    welcomeBanner: "Félicitations 🎉 Veuillez renseigner les informations de votre dApp pour donner aux utilisateurs une meilleure vue d'ensemble de votre application. Les données mises à jour apparaîtront sous peu sur la page de votre dApp.",
+    registerNow: "S'enregistrer maintenant",
+    transferableBalance: "Solde transférable",
     dappsStore: 'dApps Store',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:179]: Boutique dApps

File: src/i18n/fr/index.ts [hunk 18]
```diff
@@ -1,113 +1,551 @@
...
-    add: 'Ajouter',
-    unbond: 'Libérer',
-    unstake: 'Unstake',
-    stake: 'Staker',
+    registerDapp: "Enregistrer une dApp",
+    noDappsRegistered: "Aucun dApps enregistrées. Soyez le premier à en enregistrer un.",
+    totalStake: "Total Stake:",
+    yourStake: "Votre Stake: ",
+    add: "Ajouter",
+    unbond: 'Unbond',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:186]: Délier

File: src/i18n/fr/index.ts [hunk 19]
```diff
@@ -1,113 +1,551 @@
...
+    unstake: "Retirer",
+    stake: 'Stake',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:188]: Jalonner

File: src/i18n/fr/index.ts [hunk 20]
```diff
@@ -1,113 +1,551 @@
...
     claim: 'Claim',
```
Comment (gluneau) [inline src/i18n/fr/index.ts:189]: Réclamer

File: src/i18n/fr/index.ts [hunk 21]
```diff
@@ -1,113 +1,551 @@
...
-      "Vous avez atteint le maximum de paliers de retrait ({chunks}). Débloquez d'abord les récompenses.",
-    withdraw: 'Withdraw',
-    chunk: 'Palier',
-    chunks: 'Paliers',
-    amount: 'Montant',
-    era: 'Ère',
-    unbondedFunds: 'Fonds non engagés',
-    currentEra: 'ÈRE ACTUELLE',
-    blocksUntilNextEra: "Blocs jusqu'à la prochaine ère",
+    stakersCount: "Stakers :",
```
Comment (gluneau) [inline src/i18n/fr/index.ts:190]: Jalonneur :

**Review by gluneau — CHANGES_REQUESTED** [review]
There is still many iterations of Staking that should become "Jalonner" and the same for Unbonding as "Délier"

I will continue my review once those have been addressed.

**Comment by impelcrypto** [comment]
Hi, I've updated i18n files in [this](https://github.com/AstarNetwork/astar-apps/pull/608) PR. Please take a look at these 2 files and update the French file accordingly. Thanks!

https://github.com/AstarNetwork/astar-apps/blob/main/src/i18n/en-US/index.ts
https://github.com/AstarNetwork/astar-apps/blob/main/src/i18n/fr/index.ts

File: src/i18n/fr/index.ts (hunk 8 above)
Comment (GuiGou12358) [reply src/i18n/fr/index.ts:145]: Hello @gluneau
Je pense qu'il est préférable de garder certains termes anglais. En crypto, je pense que personne ne connais le terme jalonner ou jalonnement. Pour cette raison il est préférable de conserver le terme "staking".

**Review by GuiGou12358 — APPROVED** [review]

**Review by GuiGou12358 — APPROVED** [review]

**Comment by impelcrypto** [comment]
@gluneau I think we can close this PR since PR #694 has been merged.

**Comment by Gunit2481** [comment]
Since a French translation has been added to the portal in a different way, I close this PR.

Please refer to PR #698 for all matters related to the French version of the portal.

**Pull request closed without merging**